biggraphite/drivers/elasticsearch.py

```diff
@@ -576,10 +574,15 @@ def _search_metrics_from_components(self, glob, components, search=None):
         return False, search
 
     @GLOB_METRICS_LATENCY.time()
+    @tracing.trace
     def glob_metrics(self, glob, start_time=None, end_time=None):
         """Return a sorted list of metrics matching this glob."""
         super(_ElasticSearchAccessor, self).glob_metrics(glob, start_time, end_time)
 
+        tracing.add_attr_to_trace("glob", str(glob))
+        tracing.add_attr_to_trace("time.start", str(start_time))
+        tracing.add_attr_to_trace("time.stop", str(end_time))
```

Comment:
Naming for `time.start` and `time.stop` may be misleading. 

As parameters, they represent the bounds of the search time range. 
As I read them in the tracer, I spontaneously understand that is shows some kind of execution time.

Would you reword this?

_Same for `glob_directory_names`_